scss/style/bootstrap/_forms.switch.scss

```diff
@@ -0,0 +1,56 @@
+.custom-control.custom-switch {
```

Comment:
You can delete this file, and rename `/scss/style/bootstrap/_custom-forms.scss` to `/scss/style/bootstrap/_forms.switch.scss`, and update `/scss/style.scss` accordingly.